frontend/apps/app/package.json

```
@@ -41,6 +43,7 @@
     "lint:biome": "biome check .",
     "postinstall": "cp ../../packages/db-structure/node_modules/@ruby/prism/src/prism.wasm prism.wasm",
     "start": "next start",
+    "trigger:dev": "pnpm dlx trigger.dev@latest dev",
```

Can you fix it so that `pnpm dev` can start the dev server at once?

```suggestion
    "dev:trigger": "pnpm dlx trigger.dev@latest dev",
```